okapi/services/caches/formatters/ggz/ggzindex.tpl.php

```diff
@@ -9,14 +9,15 @@
 ?>
 <ggz xmlns="http://www.opencaching.com/xmlschemas/ggz/1/0">
     <time><?= date('c') ?></time>
-    <?php foreach ($vars['files'] as $f) { ?>
+    <?php foreach ($vars['files'] as $f) {
+    ?>
         <file>
             <name><?= $f['name'] ?></name>
             <crc><?= $f['crc32'] ?></crc>
             <time><?= date('c') ?></time>
             <?php
                 foreach ($f['caches'] as $c) {
-            ?><gch>
+                    ?><gch>
                 <code><?= $c['code'] ?></code>
```

Comment:
This looks strange, as well as the change 8 lines above. I guess the reformatting tool cannot deal well with these templates.